packages/material-ui-codemod/README.md

```diff
@@ -56,22 +56,30 @@ or
 npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/v5.0.0/moved-lab-modules.js ./src
 ```
 
-#### `textfield-variant-prop`
+#### `variant-prop`
 
-Add the TextField `variant="standard` prop when `variant` is undefined.
+Add the TextField, Select, and FormControl `variant="standard"` prop when `variant` is undefined.
 The diff should look like this:
 
 ```diff
 -<TextField value="Standard" />
 -<TextField value="Outlined" variant="outlined" />
+-<Select value="Standard" />
+-<Select value="Outlined" variant="outlined" />
+-<FormControl value="Standard" />
+-<FormControl value="Outlined" variant="outlined" />
 +<TextField value="Standard" variant="standard" />
 +<TextField value="Outlined" />
++<Select value="Standard" variant="standard" />
++<Select value="Outlined" />
++<FormControl value="Standard" variant="standard" />
++<FormControl value="Outlined" />
 ```
 
 This codemod is non-idempotent (`variant="standard"` would be added on a subsequent run, where `variant="outlined"` was removed), so should only be run once against any particular codebase.
 
 ```sh
-npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/v5.0.0/textfield-variant-prop.js ./src
+find src -name '*.js' -print | xargs npx jscodeshift -t node_modules/@material-ui/codemod/v5.0.0/variant-prop.js
```

Comment:
@petyosi can you check if I didn't break this with the rebase?

Reply:
It was.